tests/cli/commands/test_jobs.py

```diff
@@ -0,0 +1,129 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+import pytest
+from click.testing import CliRunner
+
+from airflow.cli.commands import jobs
+from airflow.jobs.scheduler_job import SchedulerJob
+from airflow.utils.session import create_session
+from airflow.utils.state import State
+from tests.test_utils.db import clear_db_jobs
+
+
+class TestCliConfigList(unittest.TestCase):
+    def setUp(self) -> None:
+        clear_db_jobs()
+        self.scheduler_job = None
+
+    def tearDown(self) -> None:
+        if self.scheduler_job and self.scheduler_job.processor_agent:
+            self.scheduler_job.processor_agent.end()
+        clear_db_jobs()
+
+    def test_should_report_success_for_one_working_scheduler(self):
+        with create_session() as session:
+            self.scheduler_job = SchedulerJob()
+            self.scheduler_job.state = State.RUNNING
+            session.add(self.scheduler_job)
+            session.commit()
+            self.scheduler_job.heartbeat()
+
+        runner = CliRunner()
+        result = runner.invoke(jobs.check, ["--job-type", "SchedulerJob"])
+        self.assertIn("Found one alive job.", result.output)
+
+    def test_should_report_success_for_one_working_scheduler_with_hostname(self):
+        with create_session() as session:
+            self.scheduler_job = SchedulerJob()
+            self.scheduler_job.state = State.RUNNING
+            self.scheduler_job.hostname = 'HOSTNAME'
+            session.add(self.scheduler_job)
+            session.commit()
+            self.scheduler_job.heartbeat()
+
+        runner = CliRunner()
+        result = runner.invoke(jobs.check, ["--job-type", "SchedulerJob", "--hostname", "HOSTNAME"])
```

Comment:
In this file, I only changed how `jobs.check` is invoked and how we check the output content / exception type.